lib/speak-tts.js

```diff
@@ -36,6 +36,7 @@ function () {
 
         var listeners = (0, _utils.isNil)(conf.listeners) ? {} : conf.listeners;
         var splitSentences = (0, _utils.isNil)(conf.splitSentences) ? true : conf.splitSentences;
+        var splitPhrases = (0, _utils.isNil)(conf.splitPhrases) ? true : conf.splitPhrases;
```

Comment:
what is this option for ?

Reply:
Sorry, looks like I messed up working with this PR and included my internal changes.  Not sure how useful this will be to the general public, but I'm trying to sync a moving picture to the TTS and I need to know when there is talking and when there is silence.  splitPhrases splits on other punctuation characters that cause a pause in the TTS (like the comma).

Reply:
alright, i'd be grateful if you could remove it. Even better, splitSentences could probably be refactored so that it can server both purposes (by passing a list of splitting chars for instance) ?

Reply:
A custom list of splitting chars would work for me and keep me from having to use a separate branch.  I'll work on that.  

Happy to remove this change, but I'm just not too familiar working with PRs...that's how I ended up with this in here in the first place :)

Reply:
I think I 'll do a massive refactor of this lib in the next month anyway, I'll include your changes. In the meantime you can use your own version and close this PR, or update the PR and I'll merge it.
It's up to you, let me know !

Reply:
ok, removed the splitPhrases option.  I'll work on making splitSentences take a custom list of splitting chars